components/Card.js

```diff
@@ -1,7 +1,44 @@
 import React from "react";
+import Card from "react-bootstrap/Card";
 
-const Card = () => {
-	return <div>Card</div>;
+const Block = ({ month, day, year, type, time, place, description }) => {
+	return (
+		// have one shadow white and one black right behind it looking like a border
+
+		<Card
+			style={{
```

Comment:
use Tailwind, also ask Raidah about how she got the black box/border working

Reply:
she did it the same way as me